Task №3

jacsmedvedev opened the pull request:
Первый коммит этой задачи. Запилил пока только сложение и вычитание.
Жду отвтета сэнсэя.

jacsmedvedev committed:
Task №3

Первый коммит этой задачи. Запилил пока только сложение и вычитание.
Жду отвтета сэнсэя.

jacsmedvedev committed:
Task №3

Переделал под обыкновенные дроби. И доделал методы умножения и
сравнения.

strazhkoe commented on src/task_from_teacher_3/FractionsMedvedev.java:
```diff
@@ -0,0 +1,55 @@
+		return _fractionPart;
+	}
+	
+	/** При параметре _degreeOfFractionalPart = 1 
```
что это за параметр?

jacsmedvedev replied:
Этот параметр должен был быть для десятичных дробей. Например если задать дробную часть   "3", то как понять это 0,3 или 0,0003 ?! Для этого и ввел этот параметр, который указывает степень дробной части.

strazhkoe replied:
Если он должен был быть и сейчас не нужен, то его надо удалить

strazhkoe commented on src/task_from_teacher_3/Main.java:
```diff
@@ -0,0 +1,84 @@
+
+	// Данный метод сумирует 2 числа типа FractionsMedvedev.
+	public static void summ(FractionsMedvedev a, FractionsMedvedev b) {
+		long commonDenominator = a.getFractionPart() * b.getFractionPart();
```
все методы арифметических операций должны быть внутри класса дроби и не должны быть static

jacsmedvedev committed:
Task №3

Перенес методы из class Main {} в class FractionsMedvedev {}.

Akros2016 commented:
+	/** При параметре _degreeOfFractionalPart = 1
>
> что это за параметр?
>
Этот параметр должен был быть для десятичных дробей. Например если задать
дробную часть "3", то как понять это 0,3 или 0,0003 ?! Для этого и ввел
этот параметр, который указывает степень дробной части.



>

strazhkoe commented on src/task_from_teacher_3/FractionsMedvedev.java:
```diff
@@ -52,4 +52,64 @@ public short getFractionPart() {
+		long commonDenominator = a.getFractionPart() * b.getFractionPart();
+		long intPartA = (commonDenominator / a.getFractionPart()) * a.getIntegerPart();
+		long intPartB = (commonDenominator / b.getFractionPart()) * b.getIntegerPart();
+		;
```
Лишняя точка с запятой

strazhkoe commented on src/task_from_teacher_3/FractionsMedvedev.java:
```diff
@@ -52,4 +52,64 @@ public short getFractionPart() {
 	}
+	
+	// Данный метод сумирует 2 числа типа FractionsMedvedev.
+	public void summ(FractionsMedvedev a, FractionsMedvedev b) {
```
Метод должен суммировать this с входящим аргументом. То есть аргумент должен быть один. Так для всех операций.

jacsmedvedev committed:
Task №3 commit 4.

Changed methods for using with a single parameter.

strazhkoe commented:
Удали из джавадока параметр, добавь джавадоки к остальным публичным методам

jacsmedvedev committed:
Task №3 commit 5

1.Deleted unnecessary parametr from javadoc to class;
2.Added javadocs to methods.

Akros2016 commented:
What's wrong ?

Akros2016 commented:
Жень, пожалуйста, глянь исправления.

Akros2016 commented:
Whyyyyyyyyy?(
